src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectSystems/VsMSBuildProjectSystem.cs

```diff
@@ -205,12 +207,14 @@ private Task AddFileCoreAsync(string path, Action addFile)
             // it into the project.
             // Other exceptions are 'web.config' and 'app.config'
             var fileName = Path.GetFileName(path);
+            var lockFileFullPath = PackagesConfigLockFileUtility.GetPackagesLockFilePath(ProjectFullPath, GetPropertyValue("NuGetLockFilePath")?.ToString(), ProjectName);
```

Comment:
`MSBuildProjectSystem.TargetFramework` reads various target framework properties, but that's the only other case which I can find. Although that surprises me, because packing a `csproj` should read some properties, right? Or a `nuspec` that has `$variables$`.